### DISCUSSION TITLE
Add accumulation of permission message when using `|`  operator

### DISCUSSION THREAD

**Discussion by rnv812** [encode/django-rest-framework#8936] [Ideas & Suggestions]
If we have expression of permissions using `|` operator the messages that custom permission classes store loses in  response details. Instead of it we have default message.

![Screenshot_20230405_183654](https://user-images.githubusercontent.com/63803175/230131578-dab7e2f3-c03e-43a5-a706-ff1b217ab580.png)

![Screenshot_20230405_185549](https://user-images.githubusercontent.com/63803175/230136556-44692499-17f3-434e-8d3e-98cb196e43ff.png) - Lost message if permission call is false

May be we should save message in `OR` class?  (to get `message` attribute of `op1` or `op2` in `__init__` call of `OR`)

I don't think it is complicated to implement this, so I could try and then create a pull request.

**Comment by auvipy** [comment]
since it is a usability improvement I am open to see the implementation
